x/evm/keeper/state_transition.go

```diff
@@ -441,6 +441,10 @@ func (k *Keeper) ApplyMessageWithConfig(ctx sdk.Context,
 	commit bool,
 	cfg *statedb.EVMConfig,
 	txConfig statedb.TxConfig,
+	// temp fix to allow for some messages to ignore min gas multiplier flag (see https://github.com/evmos/ethermint/issues/1085)
+	// to be revisited when fee mechanism is in place, currently since gasLimit is hardcoded to 1mil in our smart contracts
+	// deposit transactions have huge gas usage and fees
+	ignoreMinGasMultiplier bool,
```

Comment:
when do we ignore vs not? i see above where it's true vs. false but don't fully understand why in each case – other than `ApplyUnsignedTx` being true. But it's set to false in `traceTx` which is used in every txn type no?

I think instead of "some messages" it'd be helpful if you expanded which messages do and don't and downstream of that which methods do and don't